Client/components/index.js

```diff
@@ -1,30 +0,0 @@
-import React from 'react';
-import { QueryRenderer, graphql } from 'react-relay';
-
-import environment from './environment';
-import App from './App';
-
-class RendererHelper extends React.Component {
-  render() {
-    return (
-      <QueryRenderer
-        environment={environment}
-        query={graphql`
-          query ChangeName() {
```

Comment:
Udemy course mistakes. This namespacing is wrong and would always create errors.